packages/account/src/Constants/routes-config.js

```diff
@@ -125,9 +132,9 @@ let routesConfig;
 // For default page route if page/path is not found, must be kept at the end of routes_config array
 const route_default = { component: Page404, title: localize('Error 404') };
 
-const getRoutesConfig = () => {
+const getRoutesConfig = ({ is_deriv_crypto = false }) => {
```

Comment:
is it better for this to default to true so we hide it first then show it, rather than showing it first then hiding it?